Iobfs4proxy.podspec

```diff
@@ -40,6 +40,6 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   s.author       = { "Mike Tigas" => "https://mike.tig.as/" }
   s.platform     = :ios, '8.0'
   s.ios.deployment_target = '8.0'
-  s.source       = { :http => "https://github.com/ynd-consult-ug/iObfs/releases/download/v1/Iobfs4proxy.framework.zip" }  
+  s.source       = { :http => "https://github.com/ynd-consult-ug/iObfs/releases/download/1.0.2/Iobfs4proxy.framework.zip" }  
```

Comment:
ZIP file already uploaded -- I had to do this before merging PR to master so the proper podspec is pushed (kind of chicken -- egg situation)